tonic-web/tonic-web/src/cors.rs

```diff
@@ -0,0 +1,406 @@
+use std::sync::Arc;
+
+use http::{header, HeaderMap, HeaderValue, Method};
+use tracing::debug;
+
+use self::headers::*;
```

Comment:
Primarily to make usage clearer: `use cors::{ORIGIN, ALLOW_ORIGIN}` vs `use cors::headers::{...}` , particularly because I am re-exporting slightly different names from the ones used in `http`.  They ended up  not being needed much elsewhere, so happy to change it if you prefer.

Reply:
Yeah, if we can reduce this that would be nice.